src/quo/components/wallet/transaction_progress/component_spec.cljs

```diff
@@ -0,0 +1,101 @@
+(ns quo.components.wallet.transaction-progress.component-spec
+  (:require [quo.core :as quo]
+            [reagent.core :as reagent]
+            [test-helpers.component :as h]))
+
+(defn- get-test-data
+  [{:keys [state network]
+    :or
+    {state :pending network :mainnet}}]
+  {:title               "Title"
+   :counter             (reagent/atom 0)
+   :total-box           85
+   :accessibility-label :transaction-progress
+   :tag-name            "Doodle"
+   :tag-number          "120"
+   :epoch-number        "181,329"
+   :network             network
+   :state               state})
+
+(h/describe "Transaction Progress"
```

Comment:
Could you please also add the test where component renders without any props, to make sure it doesn't crash on such usage

Reply:
@vkjr Would be a good idea to add this to our guidelines if it is not currently there, wdyt?